tests/scale/dateTimeSpec.js

```diff
@@ -21,6 +21,27 @@
             expect(clone.domain()[1]).toEqual(end);
         });
 
+        describe('tick calculations', function() {
```

Comment:
Probably 😉

Although, this sort of behaviour is not easily black-box tested, need to find a suitable way to inject values to test the clamping behaviour